### ISSUE TITLE
TypeError

### ISSUE THREAD

**Issue by Almenon** [brandon-rhodes/uncommitted#35]
```
C:\dev>%appdata%\python\Python36\Scripts\uncommitted.exe misc
Traceback (most recent call last):
  File "C:\Users\Almenon\AppData\Roaming\python\Python36\Scripts\uncommitted-script.py", line 11, in <module>
    load_entry_point('uncommitted==2.1', 'console_scripts', 'uncommitted')()
  File "C:\Users\Almenon\AppData\Roaming\Python\Python36\site-packages\uncommitted\command.py", line 258, in main
    scan(repos, options)
  File "C:\Users\Almenon\AppData\Roaming\Python\Python36\site-packages\uncommitted\command.py", line 182, in scan
    lines, subrepos = get_status(directory, ignore_set, options)
  File "C:\Users\Almenon\AppData\Roaming\Python\Python36\site-packages\uncommitted\command.py", line 109, in status_git
    lines = [l for l in run(('git', 'status', '-s', '-b'), cwd=path)
  File "C:\Users\Almenon\AppData\Roaming\Python\Python36\site-packages\uncommitted\command.py", line 32, in run
    return check_output(command, **kw).splitlines()
  File "c:\program files\python36\lib\subprocess.py", line 356, in check_output
    **kwargs).stdout
  File "c:\program files\python36\lib\subprocess.py", line 423, in run
    with Popen(*popenargs, **kwargs) as process:
  File "c:\program files\python36\lib\subprocess.py", line 729, in __init__
    restore_signals, start_new_session)
  File "c:\program files\python36\lib\subprocess.py", line 1017, in _execute_child
    startupinfo)
TypeError: CreateProcess() argument 8 must be str or None, not bytes

C:\dev>python -V
Python 3.6.8
```

**Comment by Almenon** [comment]
Same thing when I use -l or -w flags

**Comment by brandon-rhodes** [comment]
Thank you so much for this report!

It looks like a few tweaks for Windows landed since the last release, but I was unable to test them personally. Could you try the following command?

    pip install https://github.com/brandon-rhodes/uncommitted/archive/master.zip

It should install the package from the latest version on GitHub. If the new version doesn't give the same error message as you got above, it might be verification that the patch worked and I should release. Let me know the result. Thanks!

**Comment by Almenon** [comment]
It works, thanks :)

**Comment by brandon-rhodes** [comment]
I have just released 2.2 which includes this change. Thanks again for letting me know that it didn't work for you!
